specification/cluster/get_settings/ClusterGetSettingsRequest.ts

```diff
@@ -45,7 +45,12 @@ export interface Request extends RequestBase {
      */
     flat_settings?: boolean
     /**
-     * If `true`, returns default cluster settings from the local node.
+     * If `true`, also returns default values for all other cluster settings, reflecting the values
+     * in the `elasticsearch.yml` file of one of the nodes in the cluster. If the nodes in your
+     * cluster do not all have the same values in their `elasticsearch.yml` config files then the
+     * values returned by this API may vary from invocation to invocation and may not reflect the
+     * values that Elasticsearch uses in all situations. Use the `GET _nodes/settings` API to
```

Comment:
I'm having trouble finding a `_nodes/settings` API?

Reply:
Bizarrely it's the `GET /_nodes/{nodeId}` path from `RestNodesInfoAction`, see how we compute `metricsOrNodeIds`.

Reply:
I opened https://github.com/elastic/elasticsearch-specification/issues/5094

Reply:
Ohhh, it's [`GET _nodes/{nodeId}/info/{metrics}`](https://github.com/elastic/elasticsearch/blob/58f050a38bcba4dc2aa2888516292db19224c37a/server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestNodesInfoAction.java#L50) -- I didn't have it quite right. Though not in [that documentation](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-nodes-info) you linked, anyway... 🤨